controllers/nginx/configuration.md

```diff
@@ -167,6 +167,17 @@ ingress.kubernetes.io/configuration-snippet: |
 To enable Cross-Origin Resource Sharing (CORS) in an Ingress rule add the annotation `ingress.kubernetes.io/enable-cors: "true"`. This will add a section in the server location enabling this functionality.
 For more information please check https://enable-cors.org/server_nginx.html
 
+### Server Alias
+
+To add Server Aliases to an Ingress rule add the annotation `ingress.kubernetes.io/server-alias: "<server-name_0>:<server-alias_0>;...;<server-name_n>:<server-alias_n>"`.
```

Comment:
@aledbf what if there are multiple hosts within the ingress yaml. Should the expected behavior be to added the server-alias to all hosts?